src/common/entities/task.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { Entity, PrimaryKey, Property } from '@mikro-orm/core';
+
+@Entity()
+export class Task {
+  @PrimaryKey()
+  id!: number;
+
+  @Property()
+  title!: string;
+
+  @Property({ nullable: true })
+  description?: string;
+
+  @Property({ nullable: true })
```

Comment:
```suggestion
  @Property({ nullable: true, fieldName: "due_date" })
```
Please be explicit about the db fieldName in your entity file. All @Property decorator inside entity file should contain fieldName for all attributes except for id.